installation.md

```diff
@@ -265,7 +265,7 @@ they are different.
 [Bear](https://github.com/rizsotto/Bear) is a tool to generate a
 compile_commands.json file by recording a complete build.
 
-For a `make`-based build, you can run `make clean; bear make` to generate the
+For a `make`-based build, you can run `make clean; bear -- make` to generate the
```

Comment:
Can we rather say:
```
For a `make`-based build, check Bear's [README](https://github.com/rizsotto/Bear/blob/master/README.md#how-to-use).
```

To make sure we are pointing at the source of truth, rather than getting stale again?

Reply:
Hm, I'm inclined to prefer this as-is: this doesn't seem like a thing that's going to churn a lot, and giving the common command is much more *useful* than RTFM. (We do already link to the readme)

WDYT Kadir?

Reply:
SGTM, i must've missed the link to `Bear` above.